examples/async-universal/server/server.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-console, no-use-before-define */
+
+import path from 'path'
+import Express from 'express'
+
+import webpack from 'webpack'
+import webpackDevMiddleware from 'webpack-dev-middleware'
+import webpackHotMiddleware from 'webpack-hot-middleware'
+import config from '../webpack.config'
+
+import React from 'react'
+import { renderToString } from 'react-dom/server'
+import { Provider } from 'react-redux'
+import { match, RouterContext } from 'react-router'
+
+import configureStore from '../common/store/configureStore'
+import routes from '../common/routes'
+
+const app = new Express()
+const port = 3000
+
+const compiler = webpack(config)
+app.use(webpackDevMiddleware(compiler, { noInfo: true, publicPath: config.output.publicPath }))
+app.use(webpackHotMiddleware(compiler))
+
+app.use(handleRender)
+
+function handleRender(req, res) {
+  match({
+    routes,
+    location: req.url
+  }, (error, redirectLocation, renderProps) => {
+    if (error) {
+      res.status(500).send(error.message)
+    } else if (redirectLocation) {
+      res.redirect(302, redirectLocation.pathname + redirectLocation.search)
+    } else if (renderProps) {
+
```

Comment:
Nit: unnecessary newline. Also the indentation is off (it jumps to 4 spaces while it’s 2 spaces everywhere else).
In general I think there is no need to have each two lines delimited by newlines here.